scripts/respondent-home-ui/smoke_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Why not put this in the respondent home repo? We've not do that we there other scripts so far but it's were they should be move to eventually.